state/mongodb/mongodb.go

```diff
@@ -64,6 +64,9 @@ const (
 
 	// mongodb+srv://<server>/<params>
 	connectionURIFormatWithSrv = "mongodb+srv://%s/%s"
+
+	// mongodb+srv://<server>/<params>
```

Comment:
Can you please update this comment too? Should be:

```
mongodb+srv://<username>:<password>@<server>/<params>
```

Reply:
Per @ItalyPaleAle 's comment - please update this comment